module/Frontpage/view/frontpage/frontpage/home.phtml

```diff
@@ -0,0 +1,39 @@
+<div class="jumbotron">
+    <a href="<?= $this->url('home') ?>">
+        <img src="<?= $this->basePath() ?>/img/logo.png"/>
+    </a>
+    <?php /*
+    <h1><?= sprintf($this->translate('Welcome to %sGEWIS%s'), '<span class="gewis-red">', '</span>') ?></h1>
+    <p><?= sprintf($this->translate('Welcome to the website of Study association %sGEWIS%s. This is just a bootstrap thingy for now, until the design is implemented.'), '<span class="gewis-red">', '</span>') ?></p>
+    <p><a class="btn btn-success btn-lg" href="https://github.com/GEWIS/gewisweb" target="_blank"><?php echo $this->translate('Fork the GEWIS website on GitHub') ?> &raquo;</a></p>
+ */ ?>
+</div>
```

Comment:
This bit has changed a lot in the design. See the PR on the design for this one!